Packs/SplunkPy/Integrations/SplunkPy/README.md

```diff
@@ -247,6 +247,48 @@ The following features are not supported in non-ES (Enterprise Security) Splunk.
 - Enrichment.
 - Content in the Splunk content pack (such as mappers, layout, playbooks, incident fields, and the incident type). Therefore, you will need to create your own content. See the [Cortex XSOAR Administrator’s Guide](https://docs-cortex.paloaltonetworks.com/p/XSOAR) for information.
 
+## Create KV Store
+KV Store stores your data as key-value pairs in collections.  It provides a way to save and retrieve data within your Splunk apps. The following is an example for how to create a KV Store.
+
+1. In Cortex XSOAR, create a new KV Store.
+   
+   **!splunk-kv-store-collection-create kv_store_name="\<kv_store_name\>"**
+   
+   For example:
+   
+   **!splunk-kv-store-collection-create kv_store_name=”test_kvstore”**
+   
+3. Define the fields and their type in the KV Store.
+
+   **!splunk-kv-store-collection-config kv_store_collection_name="\<kv_store_name\>" kv_store_fields="field.\<field-name\>=\<type\>,index.\<index-name\>=\<type\>,field.\<field-name-or-index\>=\<type\>,..."**
+
+   For example:
+   
+   **!splunk-kv-store-collection-config kv_store_collection_name="test_kvstore" kv_stre_fields="field.src=cidr,field.t=number,field.description=string"**
+
+   **Note:** To see the fields in Splunk, you must install the _Splunk App for Lookup File Editing_ app in Splunk.
```

Comment:
can we put here link to Splunk documentation?

Reply:
```suggestion
   **Note:** To see the fields in Splunk, you must install the _Splunk App for Lookup File Editing_ app in Splunk. For more information, see [Define a KV Store lookup in Splunk](https://docs.splunk.com/Documentation/Splunk/9.1.0/Knowledge/DefineaKVStorelookupinSplunkWeb#:~:text=Splunk%20Web%20currently,Editing%20on%20Splunkbase).
```